server/forms/interfaces/index.ts

```diff
@@ -29,6 +29,7 @@ export interface StepDefinition {
   nextStep: RulesLogic | string
   previousStep: RulesLogic | string
   validationRules: { [key: string]: Array<RulesLogic> }
+  allowedToAccess: RulesLogic | boolean
```

Comment:
It looks like the `RulesLogic` type includes `boolean` & `string` so I think the addition types on `nextStep`, `previousStep` and `allowedToAccess` are superflous